cvat/schema.yml

```diff
@@ -9320,6 +9320,28 @@ components:
     PatchedQualitySettingsRequest:
       type: object
       properties:
+        target_metric:
+          allOf:
+          - $ref: '#/components/schemas/TargetMetricEnum'
+          description: |-
+            The primary metric used for quality estimation
+
+            * `accuracy` - ACCURACY
+            * `precision` - PRECISION
+            * `recall` - RECALL
+        target_metric_threshold:
+          type: number
+          format: double
+          description: |
+            Defines the minimal quality requirements in terms of the selected target metric.
+        max_validations_per_job:
+          type: integer
+          maximum: 2147483647
+          minimum: -2147483648
```

Comment:
Is it possible to have minimum as 0?

Reply:
No, it it's checked in the serializer, but I refactored code a little bit.